### PR TITLE
[RHICOMPL-901] Store SSG and RHEL minor release support matrix

### PR DESCRIPTION
**Pull request by vkrizan**
Initial configuration of SSG support matrix.

### CODE REVIEW

File: config/supported_ssg.yaml [hunk 1]
```diff
@@ -0,0 +1,425 @@
...
+      cjis-rhel7-server:
+        policy_version: 
+      common:
+        policy_version: not versioned
+      docker-host:
+        policy_version: 
+      nist-800-171-cui:
+        policy_version: 
+      ospp-rhel7:
+        policy_version:
```
Comment (akofink) [inline config/supported_ssg.yaml:194]: We need some way to track the change in the ref_id, like:
```suggestion
      ospp-rhel7:
        old: ospp-rhel7-server
        policy_version: 
```

**Review by akofink — COMMENTED** [review]
Looks great, just a couple minor comments

File: config/supported_ssg.yaml [hunk 2]
```diff
@@ -0,0 +1,425 @@
+---
+revision: '2020-09-21'
+supported:
+  RHEL-6.6:
+    package: scap-security-guide-0.1.18-3.el6
+    brew_uri: http://download.eng.bos.redhat.com/brewroot/vol/rhel-6/packages/scap-security-guide/0.1.18/3.el6/noarch/scap-security-guide-0.1.18-3.el6.noarch.rpm
+    version: 0.1.18
+    profiles:
+      C2S:
+        policy_version:
```
[the remark this answers is not in the document]
Comment (vkrizan) [reply config/supported_ssg.yaml:10]: I can drop it. The terminology is based from the SSG spreadsheet.

File: config/supported_ssg.yaml (hunk 1 above)
Comment (vkrizan) [reply config/supported_ssg.yaml]: Added `old_names`, an array. Includes also those profiles that were merged into it.

File: config/supported_ssg.yaml [hunk 3]
```diff
@@ -0,0 +1,444 @@
...
+      common:
+        policy_version: not versioned
+      docker-host:
+        policy_version:
+      nist-800-171-cui:
+        policy_version:
+      ospp-rhel7:
+        old_names:
+          - ospp-rhel7-server
+          - nist-cl-il-al
```
Comment (akofink) [inline config/supported_ssg.yaml:196]: Is this correct? When was this identified by ref_id `nist-cl-il-al`?

**Review by akofink — APPROVED** [review]
Ok, I think we're fine to merge this. The exact fields and format may change in the future, but that's not very important for this PR. Thanks, @vkrizan!

File: config/supported_ssg.yaml (hunk 3 above)
Comment (vkrizan) [reply config/supported_ssg.yaml:196]: From the sheet: "Also nist-cl-il-al got merged into this one". We can drop the merged ones if needed.

**Pull request merged**